keras/layers/core.py

```diff
@@ -33,6 +46,25 @@ def __init__(self, **kwargs):
         if not hasattr(self, 'params'):
             self.params = []
 
+    def __call__(self, X):
```

Comment:
I used to use a method called `apply_model(model, X)` instead of `__call__` to do things like that, I just thought that `__call__` would make things cleaner and easier to use. Also `__call__` is the choice on TensorFlow, I thought this would also bring consistency.